xbmc/pvr/recordings/PVRRecordings.cpp

```diff
@@ -125,7 +125,7 @@ void CPVRRecordings::GetSubDirectories(const CPVRRecordingsPath &recParentPath,
       pFileItem->m_dateTime = current->RecordingTimeAsLocalTime();
 
       // Assume all folders are watched, we'll change the overlay later
-      pFileItem->SetOverlayImage(CGUIListItem::ICON_OVERLAY_WATCHED, false);
+      pFileItem->SetOverlayImage(CGUIListItem::ICON_OVERLAY_WATCHED);
```

Comment:
This change is a noop, as the default for parameter bOnOff of method SetOverlayImage is false. So, no change in behavior, but decrease in readability. Please revert.